app/experimenter/static/js/components/DesignForm.js

```diff
@@ -30,11 +37,17 @@ class DesignForm extends React.PureComponent {
   }
 
   isMultiPref() {
-    return this.state.data.get("is_multi_pref", this.props.isMultiPref);
+    return (
+      this.props.experimentType === TYPE_PREF &&
```

Comment:
This was an edge case I noticed where if you change the experiment type, this line would still catch and fetch the wrong serialized data so I fixed it for both multi pref and branched cases.